src/lv_message_efficient.h

```diff
@@ -18,11 +18,12 @@ namespace grpc_labview
     class LVMessageEfficient : public LVMessage
     {
     public:
-        LVMessageEfficient(std::shared_ptr<MessageMetadata> metadata) : LVMessage(metadata) {}
+        LVMessageEfficient(std::shared_ptr<MessageMetadata> metadata, int8_t* cluster) : LVMessage(metadata) { _LVClusterHandle = cluster; }
```

Comment:
nit: Use initializers

```suggestion
        LVMessageEfficient(std::shared_ptr<MessageMetadata> metadata, int8_t* cluster) : LVMessage(metadata), _LVClusterHandle(cluster) {}
```